src/camera.cc

```diff
@@ -1,62 +1,43 @@
+#define _USE_MATH_DEFINES // Needed to run in windows/visual studio
 #include "camera.h"
+#include "raytracer.h"
 #include "ray.h"
 #include "scene.h"
-#include "scene_object.h"
 // TODO: Remove when we have all point lights in vector
-#include "point_light.h"
 #include <iostream>
+#include <sstream>
 #include <random>
 
 // TODO: Place these somewhere that makes the most sense and remove some?
 const float EPSILON = 0.00001f;
-const float REFRACTION_FACTOR_OI = REFRACTION_INDEX_AIR / REFRACTION_INDEX_GLASS; // outside->in
-const float REFRACTION_FACTOR_IO = REFRACTION_INDEX_GLASS / REFRACTION_INDEX_AIR; // inside->out
-const float CRITICAL_ANGLE = asin(REFRACTION_FACTOR_OI);
-const unsigned int MAX_DEPTH = 5; // What Max depth makes sense?
+const float gamma_factor = 3.6f;
 
-std::default_random_engine generator;
-std::uniform_real_distribution<float> distribution(0, 1);
-
-Camera::Camera() {
-}
+std::default_random_engine generator3;
+std::uniform_real_distribution<float> distribution3(0, 1);
```

Comment:
Aha lol, they are both globals and the names conflict? Ok use this temp solution for now then :)